salesagility/SuiteCRM
# Fix #8856 - SugarFeed missing for Accounts module

## Description

Account Feed is added. 
[](https://github.com/salesagility/SuiteCRM/issues/8856)



## Motivation and Context

As Account is on of the most popular entities to working with in the SuiteCRM application, 
tend to think it's a good point to have feed for it. 
## How To Test This

Before: no check-box available to start stream for Accounts.

After: 
1) Administration -> Activity Stream Settings: 

2) Activate appropriate check-box: 
![image](https://user-images.githubusercontent.com/116086008/234247368-490d022b-d836-4b9e-992a-4782b4a04db7.png)

3) Create an account and check My Activity Stream dashlet: 
![image](https://user-images.githubusercontent.com/116086008/234247686-b9dd258a-9b69-46ad-a080-5a5559b0e392.png)

## Types of changes


### Final checklist
